lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -8,6 +8,13 @@
 /*
   #Layout
 */
+:root {
+  --page-width-max: 1600px;
+  --page-width-min: 1366px;
+  --content-width: 76%;
+  --sidebar-width: 100% - var(--content-width);
```

Comment:
I'm surprised this doesn't need to be a calc

Reply:
> I'm surprised this doesn't need to be a calc

In this case the units are the same, it's `100% - 76%`.